src/components/ProgressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+
+import {
+  LinearProgressBar as CoreProgressBar,
+  LinearProgressBarProps,
+} from 'wix-ui-core/linear-progress-bar';
+
+import { TPAComponentProps } from '../../types';
+
+import { st, classes } from './ProgressBar.st.css';
+
+export interface ProgressBarProps
+  extends TPAComponentProps,
+    LinearProgressBarProps {
```

Comment:
I think it might be better to just start with a subset of the core component's props, and not allow all props
since we don't support value as string, and icons at the moment.

So maybe just pass these props:
```tsx
value: number;
min: number;
max: number;
showProgressIndication: boolean;
```

Reply:
I still prefer using only these props ☝️ 
So that consumers won't pass icons and stuff like that